Index: config.m4
===================================================================
RCS file: /repository/php-src/ext/unicode/config.m4,v
retrieving revision 1.11
diff -u -r1.11 config.m4
--- config.m4	12 Jul 2007 16:08:26 -0000	1.11
+++ config.m4	17 Jan 2008 19:45:56 -0000
@@ -2,4 +2,4 @@
 dnl $Id$
 dnl
 
-PHP_NEW_EXTENSION(unicode, unicode.c locale.c unicode_iterators.c collator.c property.c constants.c transform.c, no)
+PHP_NEW_EXTENSION(unicode, unicode.c locale.c unicode_iterators.c property.c constants.c transform.c, no)
Index: config.w32
===================================================================
RCS file: /repository/php-src/ext/unicode/config.w32,v
retrieving revision 1.12
diff -u -r1.12 config.w32
--- config.w32	12 Jul 2007 16:08:26 -0000	1.12
+++ config.w32	17 Jan 2008 19:45:56 -0000
@@ -1,4 +1,4 @@
 // $Id$
 // vim:ft=javascript
 
-EXTENSION("unicode", "unicode.c unicode_iterators.c collator.c locale.c property.c constants.c transform.c", false);
+EXTENSION("unicode", "unicode.c unicode_iterators.c locale.c property.c constants.c transform.c", false);
Index: locale.c
===================================================================
RCS file: /repository/php-src/ext/unicode/locale.c,v
retrieving revision 1.11
diff -u -r1.11 locale.c
--- locale.c	1 Jan 2007 09:29:33 -0000	1.11
+++ locale.c	17 Jan 2008 19:45:56 -0000
@@ -38,6 +38,7 @@
 	*target_len = canonicalized_len;
 }
 
+#if 0
 /* {{{ proto string locale_get_default(void) U
    Returns default locale */
 PHP_FUNCTION(locale_get_default)
@@ -84,7 +85,7 @@
 	RETURN_TRUE;
 }
 /* }}} */
-
+#endif
 
 /*
  * Local variables:
Index: php_unicode.h
===================================================================
RCS file: /repository/php-src/ext/unicode/php_unicode.h,v
retrieving revision 1.15
diff -u -r1.15 php_unicode.h
--- php_unicode.h	1 Jan 2007 09:29:33 -0000	1.15
+++ php_unicode.h	17 Jan 2008 19:45:56 -0000
@@ -59,20 +59,20 @@
 #undef locale_get_default
 #undef locale_set_default
 
-PHP_FUNCTION(locale_get_default);
-PHP_FUNCTION(locale_set_default);
-PHP_FUNCTION(collator_create);
-PHP_FUNCTION(collator_compare);
-PHP_FUNCTION(collator_sort);
-PHP_FUNCTION(collator_set_strength);
-PHP_FUNCTION(collator_set_attribute);
-PHP_FUNCTION(collator_get_strength);
-PHP_FUNCTION(collator_get_attribute);
-PHP_FUNCTION(collator_get_default);
-PHP_FUNCTION(collator_set_default);
-PHP_METHOD(collator, __construct);
+//PHP_FUNCTION(locale_get_default);
+//PHP_FUNCTION(locale_set_default);
+//PHP_FUNCTION(collator_create);
+//PHP_FUNCTION(collator_compare);
+//PHP_FUNCTION(collator_sort);
+//PHP_FUNCTION(collator_set_strength);
+//PHP_FUNCTION(collator_set_attribute);
+//PHP_FUNCTION(collator_get_strength);
+//PHP_FUNCTION(collator_get_attribute);
+//PHP_FUNCTION(collator_get_default);
+//PHP_FUNCTION(collator_set_default);
+//PHP_METHOD(collator, __construct);
 
-void php_init_collation(TSRMLS_D);
+//void php_init_collation(TSRMLS_D);
 void php_register_unicode_constants(TSRMLS_D);
 
 PHPAPI zend_class_entry *u_const_ce;
Index: unicode.c
===================================================================
RCS file: /repository/php-src/ext/unicode/unicode.c,v
retrieving revision 1.45
diff -u -r1.45 unicode.c
--- unicode.c	27 Sep 2007 18:28:42 -0000	1.45
+++ unicode.c	17 Jan 2008 19:45:56 -0000
@@ -300,8 +300,8 @@
 
 /* {{{ unicode_functions[] */
 const zend_function_entry unicode_functions[] = {
-	PHP_FE(locale_get_default, NULL)
-	PHP_FE(locale_set_default, NULL)
+//	PHP_FE(locale_get_default, NULL)
+//	PHP_FE(locale_set_default, NULL)
 	PHP_FE(unicode_decode, NULL)
 	PHP_FE(unicode_semantics, NULL)
 	PHP_FE(unicode_encode, NULL)
@@ -311,10 +311,10 @@
 	PHP_FE(unicode_set_subst_char, NULL)
 	PHP_FE(unicode_get_error_mode, NULL)
 	PHP_FE(unicode_get_subst_char, NULL)
-	PHP_FE(collator_create, NULL)
-	PHP_FE(collator_compare, NULL)
-	PHP_FE(collator_get_default, NULL)
-	PHP_FE(collator_set_default, NULL)
+//	PHP_FE(collator_create, NULL)
+//	PHP_FE(collator_compare, NULL)
+//	PHP_FE(collator_get_default, NULL)
+//	PHP_FE(collator_set_default, NULL)
 
 	/* character property functions */
 	PHP_FE(char_is_lower,  NULL)
@@ -398,7 +398,7 @@
 PHP_MINIT_FUNCTION(unicode)
 {
 	php_register_unicode_iterators(TSRMLS_C);
-	php_init_collation(TSRMLS_C);
+//	php_init_collation(TSRMLS_C);
 	php_register_unicode_constants(TSRMLS_C);
 	
 	return SUCCESS;
