? diff
Index: collator.c
===================================================================
RCS file: /repository/php-src/ext/unicode/collator.c,v
retrieving revision 1.13
diff -u -r1.13 collator.c
--- collator.c	10 Apr 2007 22:44:57 -0000	1.13
+++ collator.c	2 Aug 2007 00:08:42 -0000
@@ -25,15 +25,15 @@
 
 
 zend_function_entry collator_funcs_collator[] = {
-	ZEND_ME(collator, __construct, NULL, ZEND_ACC_PUBLIC)
-	ZEND_FENTRY(getDefault, 	ZEND_FN(collator_get_default), NULL, ZEND_ACC_PUBLIC|ZEND_ACC_STATIC)
-	ZEND_FENTRY(setDefault, 	ZEND_FN(collator_set_default), NULL, ZEND_ACC_PUBLIC|ZEND_ACC_STATIC)
-	ZEND_NAMED_FE(compare, 		ZEND_FN(collator_compare), NULL)
-	ZEND_NAMED_FE(setStrength, 	ZEND_FN(collator_set_strength), NULL)
-	ZEND_NAMED_FE(setAttribute, ZEND_FN(collator_set_attribute), NULL)
-	ZEND_NAMED_FE(getStrength, 	ZEND_FN(collator_get_strength), NULL)
-	ZEND_NAMED_FE(getAttribute, ZEND_FN(collator_get_attribute), NULL)
-	ZEND_NAMED_FE(sort,			ZEND_FN(collator_sort), NULL)
+//	ZEND_ME(collator, __construct, NULL, ZEND_ACC_PUBLIC)
+//	ZEND_FENTRY(getDefault, 	ZEND_FN(collator_get_default), NULL, ZEND_ACC_PUBLIC|ZEND_ACC_STATIC)
+//	ZEND_FENTRY(setDefault, 	ZEND_FN(collator_set_default), NULL, ZEND_ACC_PUBLIC|ZEND_ACC_STATIC)
+//	ZEND_NAMED_FE(compare, 		ZEND_FN(collator_compare), NULL)
+//	ZEND_NAMED_FE(setStrength, 	ZEND_FN(collator_set_strength), NULL)
+//	ZEND_NAMED_FE(setAttribute, ZEND_FN(collator_set_attribute), NULL)
+//	ZEND_NAMED_FE(getStrength, 	ZEND_FN(collator_get_strength), NULL)
+//	ZEND_NAMED_FE(getAttribute, ZEND_FN(collator_get_attribute), NULL)
+//	ZEND_NAMED_FE(sort,			ZEND_FN(collator_sort), NULL)
 	{NULL, NULL, NULL}
 };
 
Index: config.w32
===================================================================
RCS file: /repository/php-src/ext/unicode/config.w32,v
retrieving revision 1.12
diff -u -r1.12 config.w32
--- config.w32	12 Jul 2007 16:08:26 -0000	1.12
+++ config.w32	2 Aug 2007 00:08:42 -0000
@@ -1,4 +1,4 @@
 // $Id$
 // vim:ft=javascript
 
-EXTENSION("unicode", "unicode.c unicode_iterators.c collator.c locale.c property.c constants.c transform.c", false);
+EXTENSION("unicode", "unicode.c unicode_iterators.c locale.c property.c constants.c transform.c", false);
Index: php_unicode.h
===================================================================
RCS file: /repository/php-src/ext/unicode/php_unicode.h,v
retrieving revision 1.15
diff -u -r1.15 php_unicode.h
--- php_unicode.h	1 Jan 2007 09:29:33 -0000	1.15
+++ php_unicode.h	2 Aug 2007 00:08:42 -0000
@@ -61,18 +61,18 @@
 
 PHP_FUNCTION(locale_get_default);
 PHP_FUNCTION(locale_set_default);
-PHP_FUNCTION(collator_create);
-PHP_FUNCTION(collator_compare);
-PHP_FUNCTION(collator_sort);
-PHP_FUNCTION(collator_set_strength);
-PHP_FUNCTION(collator_set_attribute);
-PHP_FUNCTION(collator_get_strength);
-PHP_FUNCTION(collator_get_attribute);
-PHP_FUNCTION(collator_get_default);
-PHP_FUNCTION(collator_set_default);
-PHP_METHOD(collator, __construct);
+//PHP_FUNCTION(collator_create);
+//PHP_FUNCTION(collator_compare);
+//PHP_FUNCTION(collator_sort);
+//PHP_FUNCTION(collator_set_strength);
+//PHP_FUNCTION(collator_set_attribute);
+//PHP_FUNCTION(collator_get_strength);
+//PHP_FUNCTION(collator_get_attribute);
+//PHP_FUNCTION(collator_get_default);
+//PHP_FUNCTION(collator_set_default);
+//PHP_METHOD(collator, __construct);
 
-void php_init_collation(TSRMLS_D);
+//void php_init_collation(TSRMLS_D);
 void php_register_unicode_constants(TSRMLS_D);
 
 PHPAPI zend_class_entry *u_const_ce;
Index: unicode.c
===================================================================
RCS file: /repository/php-src/ext/unicode/unicode.c,v
retrieving revision 1.44
diff -u -r1.44 unicode.c
--- unicode.c	12 Jul 2007 15:07:32 -0000	1.44
+++ unicode.c	2 Aug 2007 00:08:42 -0000
@@ -311,10 +311,10 @@
 	PHP_FE(unicode_set_subst_char, NULL)
 	PHP_FE(unicode_get_error_mode, NULL)
 	PHP_FE(unicode_get_subst_char, NULL)
-	PHP_FE(collator_create, NULL)
-	PHP_FE(collator_compare, NULL)
-	PHP_FE(collator_get_default, NULL)
-	PHP_FE(collator_set_default, NULL)
+//	PHP_FE(collator_create, NULL)
+//	PHP_FE(collator_compare, NULL)
+//	PHP_FE(collator_get_default, NULL)
+//	PHP_FE(collator_set_default, NULL)
 
 	/* character property functions */
 	PHP_FE(char_is_lower,  NULL)
@@ -398,7 +398,7 @@
 PHP_MINIT_FUNCTION(unicode)
 {
 	php_register_unicode_iterators(TSRMLS_C);
-	php_init_collation(TSRMLS_C);
+//	php_init_collation(TSRMLS_C);
 	php_register_unicode_constants(TSRMLS_C);
 	
 	return SUCCESS;
